### PR TITLE
Python: Add deprecated keyword to deprecated functions

### PR DESCRIPTION
**Pull request by RasmusWL**

### CODE REVIEW

**Comment by RasmusWL** [comment]
Oh look, it's green now :eyes: :heavy_check_mark:

File: python/ql/src/semmle/python/Files.qll [hunk 1]
```diff
@@ -98,16 +98,16 @@ class Folder extends Container {
...
     /** DEPRECATED: Use `getAbsolutePath` instead. */
     deprecated override string getName() {
-        folders(this, result, _)
+        result = this.getAbsolutePath()
     }
 
     /** DEPRECATED: Use `getBaseName` instead. */
     deprecated string getSimple() {
-        folders(this, _, result)
+        result = this.getAbsolutePath()
```
Comment (tausbn) [inline python/ql/src/semmle/python/Files.qll]: Are you sure about this? Are the second and third argument of `folders` really _both_ equal to `this.getAbsolutePath()`?

File: python/ql/test/2/library-tests/PointsTo/import_time/Pruned.ql [hunk 2]
```diff
@@ -7,5 +7,5 @@ from ControlFlowNode f, Location l, Context c
 
 where not PointsToInternal::reachableBlock(f.getBasicBlock(), c) and c.isImport() and
 (f.getNode() instanceof FunctionExpr or f.getNode() instanceof ClassExpr) and
-l = f.getLocation() and l.getFile().getName().matches("%test.py")
-select l.getStartLine()
\ No newline at end of file
+l = f.getLocation() and l.getFile().getShortName() = "test.py"
```
Comment (tausbn) [inline python/ql/test/2/library-tests/PointsTo/import_time/Pruned.ql]: This rewrite is more specific, as it doesn't accept e.g. files named `footest.py`.

Comment (tausbn) [reply python/ql/test/2/library-tests/PointsTo/import_time/Pruned.ql]: I guess this might be okay, since the tests still pass.

Comment (RasmusWL) [reply python/ql/test/2/library-tests/PointsTo/import_time/Pruned.ql]: I looked at the test setup, and in the cases with only one test file (`test.py`), I changed it this way.

File: python/ql/src/semmle/python/Files.qll (hunk 1 above)
Comment (RasmusWL) [reply python/ql/src/semmle/python/Files.qll]: good catch :eyes: that was some sloppy copy-paste :grimacing:

I force pushed an amended commit so this change will not be included in the final merge :+1:

**Pull request merged**